### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by JKRakuba**
### Hi Code Compilers

**OVERVIEW**
This is a To-do List Web App. It allows the user to ADD, EDIT, DELETE (REMOVE) and MARK as COMPLETED. The web app consist of a To-Do List landing page. Developed the web app using JavaScript, Node.js, Server.js, HTML and CSS.

Please review my codes and let me know if there's room for improvement. Looking forward to hearing from you.

### CODE REVIEW

**Review by PhodzoW0l7 — APPROVED** [review]

File: public/Index.html
Comment (alphineandre) [inline public/Index.html:1]: Try making the completed tasks move down when they are completed or create separate categories for them to make it look a bit more interactive.

File: public/style.css
Comment (alphineandre) [inline public/style.css:1]: Style looks good and unique

**Review by alphineandre — APPROVED** [review]
All in all the ToDoList looks good. Great work

**Review by bokangkhumalo — APPROVED** [review]
Hey @JKRAkuba, ✨ I’ve reviewed the changes, and overall, the design enhancements look great! The updated layout and color scheme make the interface much cleaner and more user-friendly. 🎨 The new design elements align well with the task at hand, and I appreciate the attention to detail in spacing and font choices. 👍 Great work on this update—approved! ✅

File: public/app.js [hunk 1]
```diff
@@ -42,6 +42,24 @@ taskForm.addEventListener('submit', function(e) {
...
+        
+        if (isCompletedA && !isCompletedB) return 1; 
+        if (!isCompletedA && isCompletedB) return -1; 
+        return 0; // if both are the same (either both completed or both not completed)
+    });
+
+    // Reattach the sorted tasks to the list
+    tasks.forEach(task => taskList.appendChild(task));
+}
+
```
Comment (alphineandre) [inline public/app.js:62]: Perfect

**Review by alphineandre — APPROVED** [review]
Nicely done

**Review by KhanyiCodes — APPROVED** [review]
ToDo List is functional, modern and unique. 
The background image is relevent.

**Review by Tepanyega — APPROVED** [review]

**Pull request merged**